data/udev-filters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Disk" : {
+	"allowed-path-patterns" : [ "ccw-*", "scsi-*", "pci-*", "platform-*" ],
+	"allowed-id-patterns" : [ "ata-*", "scsi-*", "wwn-*",
```

Comment:
`scsi-*` links are not necessarily unique. The best ones are `scsi-3*`, `scsi-2*`, `scsi-8*`. Less good: `scsi-1*`. Not recommended: `scsi-0*`, `scsi-S*`.

`wwn-*` is better, it's basically the same as `scsi-3*`.

But if you want to support legacy hardware, you should not exclude the non-unique `scsi-*` links entirely, but fall back to them if no better ones are available.

Reply:
Just saw that you were asking for a separate bug report for this. I'll see what I can do, no promises here.